prov/efa/src/rxr/rxr_pkt_entry.c

```diff
@@ -434,7 +434,7 @@ void rxr_pkt_rx_map_insert(struct rxr_ep *ep,
 	if (OFI_UNLIKELY(!entry)) {
 		FI_WARN(&rxr_prov, FI_LOG_CQ,
 			"Map entries for medium size message exhausted.\n");
-			efa_eq_write_error(&ep->util_ep, FI_ENOBUFS, -FI_ENOBUFS);
+        efa_eq_write_error(&ep->util_ep, FI_ENOBUFS, FI_EFA_ERR_RX_ENTRIES_EXHAUSTED);
```

Comment:
Why did you change the whitespace?

Reply:
I meant for it to line up with `FI_WARN()` above, but I see now it used spaces instead of tabs. :/ Will fix.